Login.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { useHistory } from 'react-router-dom';
+
+const Login = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const history = useHistory();
+
+  const handleLogin = async (e) => {
+    e.preventDefault();
+
+    try {
+      const response = await axios.post('/api/login', { email, password });
+
```

Comment:
Avoid using console.error directly. Instead, consider using a logging library that can provide more context and control over log levels. Also, consider showing the error message to the user in a friendly way.